crates/lib3h/src/transport/websocket/streams.rs

```diff
@@ -1,71 +1,33 @@
+        WsSrvAcceptResult, WsSrvMidHandshake, WsStream, WssConnectResult, WssMidHandshake,
+        WssSrvAcceptResult, WssSrvMidHandshake, WssStream, FAKE_PASS, FAKE_PKCS12,
+    },
 };
 use lib3h_protocol::DidWork;
 use std::{
-    collections::VecDeque,
     io::{Read, Write},
     sync::{Arc, Mutex},
 };
 
 use url::Url;
 
-static FAKE_PKCS12: &'static [u8] = include_bytes!("fake_key.p12");
-static FAKE_PASS: &'static str = "hello";
-
-// -- some internal types for readability -- //
-
-type TlsConnectResult<T> = Result<TlsStream<T>, native_tls::HandshakeError<T>>;
-type WsHandshakeError<T> =
-    tungstenite::handshake::HandshakeError<tungstenite::handshake::client::ClientHandshake<T>>;
-type WsConnectResult<T> =
-    Result<(WsStream<T>, tungstenite::handshake::client::Response), WsHandshakeError<T>>;
-type WsSrvHandshakeError<T> = tungstenite::handshake::HandshakeError<
-    tungstenite::handshake::server::ServerHandshake<T, tungstenite::handshake::server::NoCallback>,
->;
-type WsSrvAcceptResult<T> = Result<WsStream<T>, WsSrvHandshakeError<T>>;
-type WssHandshakeError<T> = tungstenite::handshake::HandshakeError<
-    tungstenite::handshake::client::ClientHandshake<TlsStream<T>>,
->;
-type WssConnectResult<T> =
-    Result<(WssStream<T>, tungstenite::handshake::client::Response), WssHandshakeError<T>>;
-type WssSrvHandshakeError<T> = tungstenite::handshake::HandshakeError<
-    tungstenite::handshake::server::ServerHandshake<
-        TlsStream<T>,
-        tungstenite::handshake::server::NoCallback,
-    >,
->;
-type WssSrvAcceptResult<T> = Result<WssStream<T>, WssSrvHandshakeError<T>>;
-type TlsMidHandshake<T> = native_tls::MidHandshakeTlsStream<BaseStream<T>>;
-
-type BaseStream<T> = T;
-type TlsSrvMidHandshake<T> = native_tls::MidHandshakeTlsStream<BaseStream<T>>;
-type TlsStream<T> = native_tls::TlsStream<BaseStream<T>>;
-type WsMidHandshake<T> = tungstenite::handshake::MidHandshake<tungstenite::ClientHandshake<T>>;
-type WsSrvMidHandshake<T> = tungstenite::handshake::MidHandshake<
-    tungstenite::ServerHandshake<T, tungstenite::handshake::server::NoCallback>,
->;
-type WssMidHandshake<T> =
-    tungstenite::handshake::MidHandshake<tungstenite::ClientHandshake<TlsStream<T>>>;
-type WssSrvMidHandshake<T> = tungstenite::handshake::MidHandshake<
-    tungstenite::ServerHandshake<TlsStream<T>, tungstenite::handshake::server::NoCallback>,
->;
-type WsStream<T> = tungstenite::protocol::WebSocket<T>;
-type WssStream<T> = tungstenite::protocol::WebSocket<TlsStream<T>>;
-
-type SocketMap<T> = std::collections::HashMap<String, WssInfo<T>>;
+/// how often should we send a heartbeat if we have not received msgs
+pub const DEFAULT_HEARTBEAT_MS: usize = 2000;
+
+/// when should we close a connection due to not receiving remote msgs
+pub const DEFAULT_HEARTBEAT_WAIT_MS: usize = 5000;
+
+/// a connection identifier
+pub type ConnectionId = String;
+pub type ConnectionIdRef = str;
```

Comment:
Do we need these? we deleted connection-id from the new protocol

Reply:
Yes, `StreamManager` (former `TransportWss`) still uses connection IDs. I thought about completely getting rid of those, but then decided to leave the old code as is as much as possible - and just find a clean cut. Could also change the interface of `StreamManager` to only talk about URIs. Do you think it would be worth doing so now?